src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,16 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery extends ColorSupplier {
+    public Ball getRandomBall() {
+
+        Ball ball = new Ball();
+
+        ball.setBallColor(ColorSupplier.getRandomColor());
+
+        ball.setBallNumber(new Random().nextInt(100));
+
+        return ball;
+    }
```

Comment:
There is no need to separate each of them with an empty line. <br/>
You can separate code with a blank line, if those parts of a code have different logic and it makes code cleaner or gives it better readability. But in this case, there is no use to separate them with an empty line.
```suggestion
public class Lottery extends ColorSupplier {
    public Ball getRandomBall() {
        Ball ball = new Ball();
        ball.setBallColor(ColorSupplier.getRandomColor());
        ball.setBallNumber(new Random().nextInt(100));
        return ball;
    }
```